### PR TITLE
2011 autogenerate changelog

### PR DESCRIPTION
**Pull request by IagoLast**
#2011 
---

### CODE REVIEW

File: DEVLOG.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+- Modify example
+- Update aggregation docs
+- Run all tests
+- Fix snake_case bug and add working example
+- Add aggregation docs
+- Apply proper validation messages
+- Merge branch '2005-add-aggregation' of https://github.com/CartoDB/cartodb.js into 2005-add-aggregation
+- Add validation errors
+- Add tests
+- wip
```
Comment (Jesus89) [inline DEVLOG.md]: Nice change :trollface:

Comment (IagoLast) [reply DEVLOG.md]: ?

Comment (ivanmalagon) [reply DEVLOG.md]: What are you doing here?

**Comment by Jesus89** [comment]
I have updated the script to include the devlog changes in the bump commit

**Comment by Jesus89** [comment]
Reset CHANGELOG.md file @ivanmalagon

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,13 +1,23 @@
-# CHANGELOG
+# Changelog
+
+CARTO.js is a JavaScript library that interacts with different CARTO APIs. It is part of the CARTO Engine ecosystem.
+
 All notable changes to this project will be documented in this file.
 
-The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
-and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
```
Comment (alonsogarciapablo) [inline CHANGELOG.md]: This is kind of cool ☝️

Comment (Jesus89) [reply CHANGELOG.md]: :+1: https://github.com/CartoDB/cartodb.js/commit/017188520d9daf89d2a1b5e0a2f3dccbf0f98530

**Pull request merged**